src/ui/server/helpers/summary-lists/your-business/index.ts

```diff
@@ -0,0 +1,35 @@
+import generateSummaryListRows from '../generate-summary-list-rows';
+import generateYourCompanyFields from './your-company-fields';
+import { ApplicationExporterCompany, SummaryListItemData } from '../../../../types';
+
+/**
+ * generateFields
+ * Create all fields for the insurance - Your business govukSummaryList
+ * @param {ApplicationExporterCompany} answersExporterCompany application exporterCompany object
+ * @param {number} referenceNumber application reference number
```

Comment:
`Number` same below

Reply:
do you mean ` referenceNumber: number` as below?